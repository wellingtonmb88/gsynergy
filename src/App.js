import React from "react";
import { StyleSheet, View, Platform } from "react-native";
import { Provider } from "react-redux";
import ProductListScreen from "./screens/ProductListScreen";
import store from "./store";

const marginTopDevice = Platform.OS == "iOS" ? 50 : 5;

export default class App extends React.Component {
  render() {
    return (
      <Provider store={store}>
        <View style={styles.container}>
          <ProductListScreen />
        </View>
      </Provider>
    );
  }
}

const styles = StyleSheet.create({
  container: {
    flex: 1,
    marginTop: marginTopDevice,
    padding: 5,
    backgroundColor: "#fff"
  }
});
