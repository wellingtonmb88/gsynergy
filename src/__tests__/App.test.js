import React from "react";
import App from "../App";
import { shallow } from "enzyme";

describe("Test App rendering", () => {
  it("renders App without crashing", () => {
    jest.useFakeTimers();
    const tree = shallow(<App />);
    expect(tree).toMatchSnapshot();
  });
});
