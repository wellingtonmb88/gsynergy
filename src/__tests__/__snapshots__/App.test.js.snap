// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test App rendering renders App without crashing 1`] = `
<Provider
  store={
    Object {
      "dispatch": [Function],
      "getState": [Function],
      "replaceReducer": [Function],
      "subscribe": [Function],
      Symbol(observable): [Function],
    }
  }
>
  <Component
    style={
      Object {
        "backgroundColor": "#fff",
        "flex": 1,
        "marginTop": 5,
        "padding": 5,
      }
    }
  >
    <Connect(ProductListScreen) />
  </Component>
</Provider>
`;
